index.html

```diff
@@ -26,12 +26,112 @@
             <section>
                 <h1>Rudy<br>Menekam</h1>
                 <p> Hello! I am a software developer and designer! I can help you build a product, feature or website. Taka a look at my works. If you like what you see and have a project you need coded or designed, don’t hesitate and contact me.</p>
-                <a href="#" class="button">Start collaboration</a>
+                <a href="" class="button">Start collaboration</a>
             </section>
         </div>
     </div>
     <div class="section-image">
     </div>
+    <div class="works">
+        <h1>Project</h1>
+        <div class="grid-container">
+            <div class="grid-item">
+                <img class="card_image " src="images/1.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <img class="card_image " src="images/2.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <img class="card_image " src="images/3.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
```

Comment:
- [ ] Kindly try sticking to the Figma design as one of the objectives of this project is to understand how to parse a Figma design.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/205421991-bc2c9ac2-a34b-435e-b3d6-50481b01d63c.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/205421962-efb2843e-1960-4d9f-be45-e9a24e512eb1.png" width="300">|